integration-tests/rest-tests-jaxrs/src/test/groovy/org/hawkular/metrics/rest/TenantITest.groovy

```diff
@@ -172,4 +173,160 @@ class TenantITest extends RESTTest {
 
     assertNotNull("tenantId = $tenantId, Response = $response.data", response.data.find { it.id == tenantId })
   }
+
+  @Test
```

Comment:
Can you please add tests for deleting non-existing tenants and deleting the same tenant twice?